.github/workflows/github-release.yml

```diff
@@ -0,0 +1,33 @@
+name: GitHub Release
+
+on:
+  push:
+    tags:
+      - v*
+
+jobs:
+  Publish:
+    permissions:
+      contents: write
+    runs-on: ubuntu-latest
+    if: startsWith(github.ref, 'refs/tags/v')
+    steps:
+      - name: Calculate release name
+        run: |
+          GITHUB_REF=${{ github.ref }}
+          RELEASE_NAME=${GITHUB_REF#"refs/tags/"}
+          echo "RELEASE_NAME=${RELEASE_NAME}" >> $GITHUB_ENV
+      # We'd use `github.event.head_commit.message`, but that includes the first line of the message. So we check out the code.
```

Comment:
What's wrong with the first line of the commit message?

Reply:
Nothing, it's just not part of the release notes.